ui/Makefile

```diff
@@ -49,6 +51,10 @@ $(TYPESCRIPT_TARGET): $(shell find $(TS_ROOT))
 	$(NODE_MODULES)/.bin/tsc -p $(TS_ROOT)
 	$(NODE_MODULES)/.bin/tslint -c $(TS_ROOT)/tslint.json $(addprefix -f , $(shell find $(TS_ROOT) -name *.ts -not -path *typings*))
 
+$(STYLUS_TARGET): css/*.styl
+	npm install
+	$(NODE_MODULES)/.bin/stylus $(CSS_ROOT)
```

Comment:
We should add a "generated by" comment to `app.css` (Ideally in a format that github will recognize as generated, but it doesn't look like there's anything in https://github.com/github/linguist/blob/master/lib/linguist/generated.rb that would work). See `sql/parser/Makefile`.